extension.js

```diff
@@ -14,17 +14,11 @@
 
 const TAG = 'DesktopScroller'
 
-const Gettext = imports.gettext.domain('desktop-scroller');
-const _ = Gettext.gettext;
+import {Extension, gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
 
-const Clutter = imports.gi.Clutter
-const Lang = imports.lang
-const Main = imports.ui.main
-const Meta = imports.gi.Meta
-
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-const Convenience = Me.imports.convenience;
+import Clutter from 'gi://Clutter'
+import * as Main from 'resource:///org/gnome/shell/ui/main.js'
+import Meta from 'gi://Meta'
 
 let settings = null;
 let desktopscroller = null;
```

Comment:
Can these variables converted to private variables of `Scroller` and `ScrollerExtensions`?

Reply:
This might address the following point from the review as the variable is only part of the Scroller class.

> 3. Also do this on disable:
>    settings = null;